### ISSUE TITLE
KL loss为负

### ISSUE THREAD

**Issue by Swt2000** [Dixin-s-Lab/Video-Summary-IOT#1]
`d_kl = F.kl_div(C_bc.log(), C_w)` 函数传参顺序错误，在pytorch1.10版本会出现负loss，再更高版本会出现nan
改为`d_kl = F.kl_div(F.log_softmax(C_w.double()),F.softmax(C_bc))`   #不加.double()会报类型错误

**Comment by Swt2000** [comment]
`d_kl = F.kl_div(C_bc.log(), C_w)` 函数传参顺序错误，在pytorch1.10版本会出现负loss，再更高版本会出现nan
改为`d_kl = F.kl_div(F.log_softmax(C_w.double()),F.softmax(C_bc))`   #不加.double()会报类型错误

**Comment by hhhh1138** [comment]
收到🫡，已在代码中修改，感谢指正～
